packages/fxa-auth-server/lib/senders/emails/templates/postVerify/en.ftl

```diff
@@ -1,9 +1,10 @@
-postVerify-sub-title-2 = { -product-firefox-account } confirmed. You’re almost there.
-postVerify-title = Next sync between your devices!
-postVerify-description = Sync privately keeps your bookmarks, passwords and other { -brand-firefox } data the same across all your devices.
-postVerify-subject-2 = Account confirmed. Next, sync another device to finish setup
-postVerify-setup = Set up next device
-postVerify-action = Set up next device
+postVerify-sub-title-3 = We're delighted to see you!
+postVerify-title-2 = Want to see the same tab on two devices?
+postVerify-description-2 = It's easy! Just install { -brand-firefox } on another device and log in to sync. It's like magic!
+postVerify-sub-description = (Psst...It also means you can get your bookmarks, passwords, and other { -brand-firefox } data everywhere you're signed in.)
```

Comment:
```suggestion
postVerify-sub-description = (Psst...It also means you can get your bookmarks, passwords, and other { -brand-firefox } data everywhere you’re signed in.)
```

Reply:
@bcolsson @flodolo Should this be "Psst..." or ellipsis like "Psst…"?

Reply:
```suggestion
postVerify-sub-description = (Psst… It also means you can get your bookmarks, passwords, and other { -brand-firefox } data everywhere you’re signed in.)
```
Yep, ellipsis is correct. (Also added a space after it.)